### ISSUE TITLE
[BUG] [KMM] Cannot use in `iosCommon`

### ISSUE THREAD

**Issue by hoc081098** [InsertKoinIO/koin-annotations#41]
**Describe the bug**
```
@JvmMultifileClass can only be used in common module sources
@JvmName can only be used in common module sources
```

**Comment by javichaques** [comment]
I have the same issue, have you been able to solve it @hoc081098 ?

**Comment by hoc081098** [comment]
> I have the same issue, have you been able to solve it @hoc081098 ?

I'm writing module manually 😂.

**Comment by arnaudgiuliani** [comment]
Header geenration is fixed back in `1.0.2` 👍
